spec-main/chromium-spec.ts

```diff
@@ -267,6 +268,38 @@ describe('command line switches', () => {
     it('should not set an invalid locale', (done) => testLocale('asdfkl', currentLocale, done));
   });
 
+  describe('--remote-debugging-pipe switch', () => {
+    it('should expose CDP via pipe', async () => {
```

Comment:
@MarshallOfSound : Now that I have two tests here, on my Linux box I can see Electron app started in a headful manner for a brief moment. Is that expected?

Reply:
@pavelfeldman Reading through these tests, yes.  They spawn the electron executable with no app argument so it will launch the default app (which will open a window)